realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -1196,6 +1198,9 @@ private void checkMaxDepth(int maxDepth) {
         if (!realmObject.isValid()) {
             throw new IllegalArgumentException("RealmObject is not valid, so it cannot be copied.");
         }
+        if (realmObject instanceof DynamicRealmObject) {
+            throw new IllegalArgumentException("DynamicRealmObjects cannot be copied from Realm");
```

Comment:
missing `.` at the end of exception message.
`DynamicRealmObjects` -> `DynamicRealmObject`